js/utils.js

```diff
@@ -2299,7 +2302,8 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
       appendUrlQuery: appendUrlQuery
     },
     Date: {
-      moment: getMomentDate
+      moment: getMomentDate,
+      LOCALE_FORMATS
```

Comment:
I'd rather use a method, e.g. `this.Utils.Date.getLocaleFormat(format)` with `format` would be `date` or `time`. Functions are easier to upgrade should it be necessary.